save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/basic/ExecutionStatistics.kt

```diff
@@ -55,14 +55,11 @@ fun executionStatistics(classes: String = "") = fc<ExecutionStatisticsProps> { p
 /**
  * A component which displays a GIF if tests not found
  *
+ * @param count tests for execution
  * @return a functional react component
  */
-fun executionTestsNotFound() = fc<ExecutionStatisticsProps> { props ->
-    val totalTests = props.executionDto?.run {
-        runningTests + passedTests + failedTests + skippedTests
-    } ?: 0
-
-    if (totalTests == 0L) {
+fun executionTestsNotFound(count: Int?) = fc<ExecutionStatisticsProps> {
+    if (count == 0) {
```

Comment:
Are we doing it as a workaround, while coutners in execution are unreliable?

Reply:
Counters in execution do not take into account many statuses

Reply:
Then we should also use this value instead of `totalTests` in `fun executionStatistics`, because the same problem applies there: even if all four counters are correct, their sum may be different from what's actually displayed in the table, because we currently have 7 statuses and only 4 counters

Reply:
Fixed